### PR TITLE
theme: add aqua theme

### PR DESCRIPTION
**Pull request by pride7**

### CODE REVIEW

File: themes/themes.typ [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #import "simple.typ"
 #import "metropolis.typ"
 #import "dewdrop.typ"
-#import "university.typ"
\ No newline at end of file
+#import "university.typ"
+#import "blue.typ"
```
Comment (OrangeX4) [inline themes/themes.typ]: You should look for a more creative name, using a commonly used color (one of the three primary colors) to name a theme is not so good.

File: themes/blue.typ [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+      )
+    )
+  })
+}
+
+#let register(
+  aspect-ratio: "16-9",
+  self,
+) = {
+  self.title_color = rgb(0, 63, 136)
```
Comment (OrangeX4) [inline themes/blue.typ]: You should find a color theme for your theme, use `self.methods.colors` to register the color theme and use it, like `https://touying-typ.github.io/touying/docs/build-your-own-theme#color-theme`

File: themes/blue.typ [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+  self.author_color = rgb(33,89,165)
+  self.date_color = rgb(33,89,165)
+  self.icon_color = rgb(242,244,248)
+  self.slide_title = []
```
Comment (OrangeX4) [inline themes/blue.typ]: You shouldn't use underscores to name variables, you can use 'kebab-case', and you're better using the form 'self.themeprefix-title'.

File: themes/blue.typ [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+  self.section_num = counter("section")
+  self.background = {
+  place(
+    left+top,
```
Comment (OrangeX4) [inline themes/blue.typ]: I prefer to add a space before and after the plus sign, but it's not mandatory, you can do as you like.

File: themes/blue.typ [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+    dx: -15pt,
+    dy: -26pt,
+    circle(
+    radius: 40pt,
+    fill: self.title_color,
```
Comment (OrangeX4) [inline themes/blue.typ]: Incorrect indentation, and you can consider compressing the place function to one line to reduce the number of lines of code.

File: themes/blue.typ [hunk 6]
```diff
@@ -0,0 +1,365 @@
+#import "../utils/utils.typ"
+#import "../utils/states.typ"
+#import "../utils/components.typ"
+
+#let title-slide(self: none) = {
+  self = utils.empty-page(self)
+  self.page-args = self.page-args + (
+    margin: (top: 30%, left:17%, right:17%, bottom:0%),
+    background: self.background,
```
Comment (OrangeX4) [inline themes/blue.typ]: Have you ever considered the background display of a "4-3" aspect-ratio?

**Comment by OrangeX4** [comment]
Overall, I love this theme, just need a little fix, and I'll add it to Touying's built-in theme, thank you!

File: examples/blue.typ [hunk 7]
```diff
@@ -0,0 +1,62 @@
+#import "../lib.typ": *
+
+#let s = themes.blue.register(
+  s, aspect-ratio: "16-9"
+)
+#let s = (s.methods.info)(
+  self: s, 
+  title: [Typst Beamer如何做？#linebreak()这是一个新手教程],
```
Comment (OrangeX4) [inline examples/blue.typ]: You should use English examples.

File: themes/aqua.typ [hunk 8]
```diff
@@ -0,0 +1,308 @@
...
+    dx: -5%, 
+    [#counter(page).display("1")]
+  )
+}
+
+  let header(self)={
+  place(
+    center+top,
+    dy: 45%,
+    image("./images/header.png", width: 100%)
```
Comment (OrangeX4) [inline themes/aqua.typ]: Why is there a `header.png` here?

Comment (pride7) [reply themes/aqua.typ]: Oh, it must be because I forgot to delete that part, I'll fix it right away.

**Review by pride7 — COMMENTED** [review]
Thank you!

**Review by OrangeX4 — APPROVED** [review]

**Pull request merged**